shlr/capstone.sh

```diff
@@ -1,58 +1,84 @@
-#!/bin/sh
+#! /bin/sh
 CS_URL="$1" # url
 CS_BRA="$2" # branch name
 CS_TIP="$3" # tip commit
 CS_REV="$4" # revert
+CS_DEPTH_CLONE=10
 
-clone_capstone() {
-	if [ ! -d capstone ]; then 
-		git clone -q -b "${CS_BRA}" --depth 10 "${CS_URL}" capstone || exit 1
-	fi
-	cd capstone || exit 1
-	if [ -n "${CS_REV}" ]; then
-		HEAD="`git log|grep ^commit | head -n2|tail -n1 | awk '{print $2}'`"
-	else
-		HEAD="`git log|head -n1 | awk '{print $2}'`"
-	fi
-	BRANCH="`git branch | grep '*' | cut -d'*' -f2 | cut -d' ' -f2`"
-
-	if [ "${HEAD}" = "${CS_TIP}" ]; then
-		echo "[capstone] Already in TIP, no need to update from git"
-		exit 0
-	fi
+fatal_msg() {
+	>&2 printf '[capstone] %s\n' "$1"
```

Comment:
Dafuq?

Reply:
? See [this](https://stackoverflow.com/a/23550347/5456794). But it's just a matter of style, so I'll change it as you want.